goodkey/good_key.go

```diff
@@ -146,10 +146,10 @@ func (policy *KeyPolicy) GoodKey(ctx context.Context, key crypto.PublicKey) erro
 		if err != nil {
 			return badKey("%w", err)
 		}
-		exists, err := policy.dbCheck(ctx, &sapb.KeyBlockedRequest{KeyHash: digest[:]})
+		exists, err := policy.dbCheck(ctx, digest[:])
```

Comment:
Now that BlockedKeyCheckFunc is much more generic, I'd consider renaming `policy.dbCheck` to `policy.blockedCheck` or something like that.

Reply:
Updated, thanks!